cmd/participation/remove.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Removes a OSN from an existing channel.
+func remove(osn, tlsDir, channelID string) {
+	client := httpClient(tlsDir)
+	url := fmt.Sprintf("https://%s/participation/v1/channels/%s", osn, channelID)
+
+	req, err := http.NewRequest(http.MethodDelete, url, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	resp, err := client.Do(req)
+	if err != nil {
+		panic(err)
+	}
+
+	if resp.StatusCode != http.StatusNoContent {
+		panic(fmt.Sprintf("Removal of channel %s failed with status %d", channelID, resp.StatusCode))
+	}
```

Comment:
In case of a failed request that returns an error status code, the response contains an informative error message. It is better to print it as well. This relates to all commands. See 
https://docs.google.com/document/d/1kYoicS_Z59gNvmuuB7VFZ6WsOL31C34dWMeOFQiIygA/edit?usp=sharing

Reply:
Refactored the code. We now print the status and error generated by the server for all commands.